tests/amd/amd.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect, CKEDITOR */
```

Comment:
Heh... I've been already thinking about this too. I think that the easiest (thus, doable) solutions are:

* Define little in the global configs and use comments extensively. The thing is that the global config of the ckeditor5-core module should for example contain `CKEDITOR` because it should be available in most of the files, but global config of the ckeditor5 module does not have to, because many files in this repository are related to grunt tasks. The rest will need to be defined in comments, which is not that bad, because you only need to do this when you start using some variable.
* Define a lot in the global configs. We can add most of the stuff that can be available somewhere in tests or dev code to the global config and we won't need to add so many comments. This makes the check less precise, but it's still good.

In CKEditor 4 we use now the second approach and it's ok. For instance, you never use `it` or `describe` in the dev code, so you don't do mistakes related to them.